src/shared/seccomp-util.c

```diff
@@ -1063,7 +1063,7 @@ int seccomp_load_syscall_filter_set_raw(uint32_t default_action, Hashmap* set, u
         uint32_t arch;
         int r;
 
-        /* Similar to seccomp_load_syscall_filter_set(), but takes a raw Set* of syscalls, instead of a
+        /* Similar to seccomp_load_syscall_filter_set(), but takes a raw Hashmap* of syscalls, instead of a
```

Comment:
Hmm, maybe also rename `set` to `map` or `filter` or something? The variable name is misleading too…